Bigtable/src/BigtableClient.php

```diff
@@ -0,0 +1,200 @@
+     *           credentials .json file retrieved from the Google Developers
+     *           Console.
+     *     @type float $requestTimeout Seconds to wait before timing out the
+     *           request. **Defaults to** `0` with REST and `60` with gRPC.
+     *     @type int $retries Number of retries for a failed request.
+     *           **Defaults to** `3`.
+     * }
+     * @throws GoogleException If the gRPC extension is not enabled.
+     */
+    public function __construct(array $config = [])
+    {
+        $this->requireGrpc();
+        $config += [
+            'scopes' => [
+                self::FULL_CONTROL_SCOPE,
+                self::ADMIN_SCOPE
+            ],
+            'projectIdRequired' => true
+        ];
+        $this->connection = new Grpc($this->configureAuthentication($config));
+        $this->setLroProperties(new LongRunningConnection($this->connection), [
+            [
+                'typeUrl' => 'type.googleapis.com/google.bigtable.admin.instance.v2.UpdateInstanceMetadata',
+                'callable' => function ($instance) {
+                    $name = InstanceAdminClient::parseName($instance['name'])['instance'];
+                    return $this->instance($name, $instance);
+                }
+            ], [
+                'typeUrl' => 'type.googleapis.com/google.bigtable.admin.instance.v2.CreateInstanceMetadata',
+                'callable' => function ($instance) {
+                    $name = InstanceAdminClient::parseName($instance['name'])['instance'];
+                    return $this->instance($name, $instance);
+                }
+            ]
+        ]);
+    }
+
+    /**
+     * Create a new instance.
+     *
+     * Example:
+     * ```
+     * $operation = $bigtable->createInstance('my-instance');
+     * ```
+     *
+     * @codingStandardsIgnoreStart
+     * @see https://cloud.google.com/bigtable/docs/reference/admin/rpc/google.bigtable.admin.v2#google.bigtable.admin.v2.CreateInstanceRequest CreateInstanceRequest
+     *
+     * @param string $instanceId The instance ID.
+     * @param array $options [optional] {
+     *     Configuration options
+     *
+     *     @type string $displayName **Defaults to** the value of $instanceId.
+     *     @type int $instanceType Possible values are represented by the following constants: `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_PRODUCTION`, `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_DEVELOPMENT` and `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_UNSPECIFIED`.
+     *           **Defaults to** using `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_UNSPECIFIED`.
+     *     @type array $labels as key/value pair ['foo' => 'bar']. For more information, see
+     *           [Using labels to organize Google Cloud Platform resources](https://cloudplatform.googleblog.com/2015/10/using-labels-to-organize-Google-Cloud-Platform-resources.html).
+     *     @type array $clusters {
```

Comment:
I see that in the latest change, we're back to using an array. If there is a reason you're not in favor of adding a `Cluster` class, I'm open to that, but let's have a discussion about the reasons both for and against.

Reply:
Agreed. A discussion would be great.

Reply:
@dwsupplee and @jdpedrie when are you available for discussion? Please let us know.

Reply:
@AVaksman Is there any reason we wouldn't be able to discuss this over GitHub? @jdpedrie outlined a proposal and we haven't heard anything back on it.

Reply:
To follow up here: After some offline discussions we are good with using an array to represent a cluster. Afterwards, we plan to add a factory method which will help make creating the required data structure easier on users.